cupy/cuda/cusparse.pxd

```diff
@@ -31,6 +31,23 @@ cdef extern from *:
     ctypedef int cusparseDirection_t
     ctypedef int cusparseSolvePolicy_t
 
+    ctypedef int IndexType 'cusparseIndexType_t'
```

Comment:
Maybe some comment here stating that the next declarations are for the generic API would help :)

Reply:
Thanks, will add the comment here.